lms/djangoapps/courseware/views/views.py

```diff
@@ -510,11 +510,14 @@ def course_about(request, course_id):
         ecommerce_bulk_checkout_link = ''
         professional_mode = None
         ecomm_service = EcommerceService()
-        if ecomm_service.is_enabled(request.user) and (
-                CourseMode.PROFESSIONAL in modes or CourseMode.NO_ID_PROFESSIONAL_MODE in modes
-        ):
-            professional_mode = modes.get(CourseMode.PROFESSIONAL, '') or \
+        if (ecomm_service.is_enabled(request.user) and
+            (CourseMode.HONOR in modes or
+             CourseMode.PROFESSIONAL in modes or
+             CourseMode.NO_ID_PROFESSIONAL_MODE in modes)):
+            professional_mode = (
+                modes.get(CourseMode.HONOR, '') or
                 modes.get(CourseMode.NO_ID_PROFESSIONAL_MODE, '')
```

Comment:
here probably `modes.get(CourseMode.PROFESSIONAL, '')` is needed as well, (not sure how important it is but not to accidentally break something)